Ginger/Ginger/GeneralValidationRules/MaxLengthValidationRule.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+
+namespace Amdocs.Ginger.ValidationRules
+{
+    public class MaxLengthValidationRule : ValidationRule
+    {       
+        string mMessage;
+        string Message
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(mMessage))
+                {
+                    return "Maximum Length allowed is {0}";
+                }
+                else
+                {
+                    return mMessage;
+                }
+            }
+            set
+            {
+                mMessage = value;
+            }
+        }
+
+        int mMaxLength;
+
+        public MaxLengthValidationRule()
```

Comment:
Do we need empty constructor?

Reply:
No, i'll remove it from all validation rule if not needed.